### ISSUE TITLE
Confusing documentation: report it here!

### ISSUE THREAD

**Issue by Zac-HD** [HypothesisWorks/hypothesis#1514]
This meta-issue is for reporting all confusing or out-of-date documentation / install instructions / etc at the PyCon Australia sprints (see #1513 for the main meta-issue).

**Comment by massey101** [comment]
Personally I feel that "how do I build this thing" is the first thing I want to know when I end up at a github site. Maybe the README can point us towards CONTRIBUTING for that and then it can just stay where it is.

**Comment by massey101** [comment]
Also:

jack@jackspc:~/hypothesis$ grep -ri vagrant
Vagrantfile:# This is a trivial Vagrantfile designed to simplify development of Hypothesis on Windows,
Vagrantfile:# up a suitable local VM for use with vagrant ssh. You should then use the Makefile from within
Vagrantfile:if [ ! $(grep -q 'cd /vagrant' $HOME/.bashrc) ]; then
Vagrantfile:    echo 'cd /vagrant' >> $HOME/.bashrc
Vagrantfile:cd /vagrant/
Vagrantfile:Vagrant.configure(2) do |config|
.gitignore:.vagrant

Apparently there is no documentation on using vagrant.

**Comment by 00willo** [comment]
Yeah agreed, I couldn't even find the install stuff without doing a text search for "install".  Links to both getting started for users and contributors would be great straight off the main README.

**Comment by Zac-HD** [comment]
Yeah... turns out that this needs a fair bit of work sorry!  I've written up working instructions on #1513 and will fix all the other ones after the sprints :cry:

**Comment by BexDunn** [comment]
Ok, this is my first read-through and I'm working my way through the explanation of hypothesis in the quickstart guide. https://hypothesis.readthedocs.io/en/latest/quickstart.html 

I found the example (image) hard to understand on the first read-through (only worked it out while writing this - I'm a bit slow) - maybe we could edit for clarity and explain that you import the example and stick it in the decorator above the function (if this was really really obvious to everyone but me, I'm sorry, I used a decorator for the first time on Thursday). 

Also possibly expand/clarify this sentence: "You don’t have to do this, but it can be useful both for clarity purposes and for reliably hitting hard to find examples. "

![image](https://user-images.githubusercontent.com/20038594/44636068-d70b0600-a9ec-11e8-9881-2e85404b3b70.png)

**Comment by sponster-au** [comment]
https://hypothesis.readthedocs.io/en/latest/details.html#hypothesis.given
says "The @given decorator may be used to specify which arguments of a function should be parametrized over. You can use either positional or keyword arguments or a mixture of the two."

This is not true; if you use a mixture of positional and keyword arguments to @given, it raises:

hypothesis.errors.InvalidArgument: cannot mix positional and keyword arguments to @given

**Comment by piccolbo** [comment]
As far as randomized testing here is one clear reference to it: https://hypothesis.readthedocs.io/en/latest/reproducing.html

Hypothesis is described as randomized testing in a number of other documents but I guess that's outside the scope of this issue or our control.
